File: 20140117/fizzbuzz.py
def is_dividable(dividee, divider):
    return dividee % divider == 0

def is_fizz(num):
    return is_dividable(num, 3)

def is_buzz(num):
    return is_dividable(num, 5)

def fizzbuzz(num):
    if (is_fizz(num) and is_buzz(num)):
        return "FizzBuzz"
    elif (is_fizz(num)):
        return "Fizz"
    elif (is_buzz(num)):
        return "Buzz"
    else:
        return num

if __name__ == '__main__':
    for i in range(1, 101):
        print fizzbuzz(i)

File: 20140122/fizzbuzz.js
var console = {};
console.log = function (str){
  print(str);
};

function is_dividable(dividee, divider){
  return (dividee % divider == 0);
}
function is_fizz(num){
  return is_dividable(num, 3);
}

function is_buzz(num){
  return is_dividable(num, 5);
}

function fizzbuzz(num){
  if (is_fizz(num) && is_buzz(num))
    console.log("FizzBuzz");
  else if (is_fizz(num))
    console.log("Fizz");
  else if (is_buzz(num))
    console.log("Buzz");
  else
    console.log(num);
}

for (var i = 0; i < 100; i++){
  fizzbuzz(i+1);
}